src/contracts/util.ts

```diff
@@ -1,2 +1,2 @@
-export const rpcUrl = import.meta.env.PUBLIC_SOROBAN_RPC_URL ?? "http://localhost:8000/"
+export const rpcUrl = import.meta.env.PUBLIC_SOROBAN_RPC_URL ?? "http://localhost:8000/soroban/rpc"
```

Comment:
For a while you've been able to use this shorter URL now.
```suggestion
export const rpcUrl = import.meta.env.PUBLIC_SOROBAN_RPC_URL ?? "http://localhost:8000/rpc"
```

Reply:
good to know!